### ISSUE TITLE
NV41: stuck on version 1.6.0

### ISSUE THREAD

**Issue by rekado** [Dasharo/dasharo-issues#633]
**Dasharo version**
v1.6.0

**Dasharo variant**
NV4x 12th Gen

**Question text**
I've just upgraded my Novacustom NV41 to version 1.6.0 by booting into the tools suite via USB stick.  I see that version 1.7.1 should be available, but when I select option 5 to upgrade it only ever looks for version 1.6.0 and concludes correctly that this version has already been installed.  Should I be able to get version 1.7.1 for this laptop?  Do I need to do something else to install it other than booting into DTS 1.2.8 via USB?

**Comment by rekado** [comment]
The version was bumped to 1.7.1 with DTS 1.2.11, but the latest version offered on https://docs.dasharo.com/dasharo-tools-suite/releases/ is 1.2.8.  I'll download 1.2.11 and create a new bootable USB thumb drive from it to see if that offers me to install 1.7.1.

**Comment by wessel-novacustom** [comment]
Hi and thank you for submitting your ticket!

DTS 1.2.11 should indeed help you to upgrade to v1.7.1. Please follow these instructions to do so: https://docs.dasharo.com/unified/novacustom/firmware-update/#updating-older-versions

@macpijan The actual 'bug' or issue here is that the [Dasharo Tools Suite releases page](https://docs.dasharo.com/dasharo-tools-suite/releases/) was not updated since v1.2.8. If you can, please solve this.

**Comment by rekado** [comment]
Yes, I fetched DTS 1.2.11 (by blindly changing the URL), and with that version I was able to upgrade to 1.7.1.  Listing versions beyond 1.2.8 on the release page would in fact solve this issue.

**Comment by wessel-novacustom** [comment]
Perfect! @macpijan should we close the issue or should we wait until the releases have been updated on the mentioned page?

**Comment by macpijan** [comment]
It is there

**Comment by wessel-novacustom** [comment]
@macpijan Perfect!

@rekado I think the issue can be closed?

**Comment by rekado** [comment]
yes, all good now.  Thanks!
